### ISSUE TITLE
Route /talent/{talentId} not returning any results.

### ISSUE THREAD

**Issue by networkscraper** [networkscraper/networkscraper#3]
Debug output:
```
array (size=2)
  0 => 
    array (size=3)
      'query' => string 'select * from `talent` where `talent_id` = ? limit 1' (length=52)
      'bindings' => 
        array (size=1)
          0 => string 'bret_hart' (length=9)
      'time' => float 0.65
  1 => 
    array (size=3)
      'query' => string 'select `milestones`.*, `milestones_talent`.`talent_id` as `pivot_talent_id`, `milestones_talent`.`milestone_id` as `pivot_milestone_id` from `milestones` inner join `milestones_talent` on `milestones`.`id` = `milestones_talent`.`milestone_id` where `milestones_talent`.`talent_id` in (?)' (length=287)
      'bindings' => 
        array (size=1)
          0 => string 'bret_hart' (length=9)
      'time' => float 0.49
```

Something wrong with the 2nd query

**Comment by networkscraper** [comment]
Problem is this part of the query:

```
`milestones` inner join `milestones_talent` on `milestones`.`id` = `milestones_talent`.`milestone_id` 
```

Instead should be:

```
`milestones` inner join `milestones_talent` on `milestones`.`milestone_id` = `milestones_talent`.`milestone_id`
```

**Comment by networkscraper** [comment]
Resolved in ae8c2d95c3f45e0930266d83d5a881671c011391
